### DISCUSSION TITLE
怎么测试自己修改的包描述文件

### DISCUSSION THREAD

**Discussion by heheda123123** [xmake-io/xmake#2926] [General]
提了个issue
https://github.com/xmake-io/xmake-repo/issues/1535
我想自己修改 https://github.com/xmake-io/xmake-repo/blob/master/packages/l/libhv/xmake.lua 这个lua文件，加上链接advapi32.lib看行不行，但是具体怎么测试看了半天没看懂，
![image](https://user-images.githubusercontent.com/97490782/196017134-f524aa82-d6e2-42dd-9c9b-75c01b3c9bdf.png)

能提供一个详细的步骤吗？
比如在本地新建这个xmake.lua文件，然后执行什么命令尝试构建看是否成功

**Comment by waruqi** [comment]
进到 xmake-repo 目录

**Reply by heheda123123** [reply]
这样吗？
1 先把 xmake-repo clone到本地，然后修改对应的包，比如我上面提到的libhv的xmake.lua
2 然后执行 xmake l scripts/test.lua --shallow libhv

**Reply by waruqi** [reply]
嗯 你试下不就知道了

**Comment by heheda123123** [comment]
ok， 解决了
https://github.com/xmake-io/xmake-repo/pull/1536
